### PR TITLE
fix receiving all requests to student circles in the organization is a sparate request

### PR DESCRIPTION
**Pull request by tsar123**

### CODE REVIEW

File: open_schools_platform/organization_management/organizations/urls.py [hunk 1]
```diff
@@ -20,7 +20,4 @@
     path('/<uuid:pk>/students/export', OrganizationStudentProfilesExportApi.as_view(),
          name='export-organization-students'),
     path('/<uuid:pk>/analytics', GetAnalytics.as_view(), name='analytics'),
-    path('/<uuid:organization_id>/student-profile/<uuid:student_profile_id>/circles', QueriesCirclesOrganizationStudent.as_view(), name='queries-organization-student-profile'),
-    path('/<uuid:pk>/teachers', OrganizationTeachersListApi.as_view(), name='teachers-list'),
-    path('/teachers/<uuid:pk>', GetTeacherApi.as_view(), name='get-teacher')
-]
```
Comment (Feazom) [inline open_schools_platform/organization_management/organizations/urls.py:26]: If you remove endpoints, you should either change or remove the tests for these URLs.

I'm out of context, is it really necessary to remove those endpoints?

Comment (tsar123) [reply open_schools_platform/organization_management/organizations/urls.py:26]: I didn't write it and was afraid that I would accidentally appropriate it. do I need to leave this code and tests?

File: open_schools_platform/organization_management/organizations/views.py [hunk 2]
```diff
@@ -277,26 +276,26 @@ def get(self, request, pk):
         return Response({"analytics": AnalyticsSerializer(count_queries_by_statuses(queries)).data}, status=200)
 
 
-class QueriesCirclesOrganizationStudent(ApiAuthMixin, ListAPIView):
+class QueriesOrganizationStudent(ApiAuthMixin, ListAPIView):
```
Comment (Feazom) [inline open_schools_platform/organization_management/organizations/views.py:279]: I think OrganizationStudentProfileQueriesApi is better

Comment (tsar123) [reply open_schools_platform/organization_management/organizations/views.py]: ok

File: open_schools_platform/organization_management/organizations/urls.py [hunk 3]
```diff
@@ -20,7 +20,8 @@
     path('/<uuid:pk>/students/export', OrganizationStudentProfilesExportApi.as_view(),
          name='export-organization-students'),
     path('/<uuid:pk>/analytics', GetAnalytics.as_view(), name='analytics'),
-    path('/<uuid:organization_id>/student-profile/<uuid:student_profile_id>/circles', QueriesCirclesOrganizationStudent.as_view(), name='queries-organization-student-profile'),
+    path('/<uuid:organization>/student-profile/<uuid:student_profile>/queries',
```
Comment (Feazom) [inline open_schools_platform/organization_management/organizations/urls.py:23]: .../student-profiles/<uuid...

File: open_schools_platform/organization_management/organizations/views.py [hunk 4]
```diff
@@ -277,26 +276,26 @@ def get(self, request, pk):
...
+            filters={"id": str(organization)},
             user=request.user,
             empty_exception=True
         )
         queries = get_queries(
-            filters={"sender_id": str(student_profile_id),
+            filters={"sender_id": str(student_profile),
                      "recipient_ids": form_ids_string_from_queryset(organization.circles.values())}
         )
         return Response({"results": StudentProfileQuerySerializer(queries, many=True).data}, status=200)
```
Comment (Feazom) [inline open_schools_platform/organization_management/organizations/views.py:301]: It is documented in swagger that the response is paginated, but your response is just an array of queries. Maybe you should use get_paginated_response function

**Review by Feazom — CHANGES_REQUESTED** [review]

File: open_schools_platform/organization_management/organizations/urls.py (hunk 3 above)
Comment (tsar123) [reply open_schools_platform/organization_management/organizations/urls.py:23]: ok

File: open_schools_platform/organization_management/organizations/views.py (hunk 4 above)
Comment (tsar123) [reply open_schools_platform/organization_management/organizations/views.py:301]: what do you mean? do "pagination_class = get_paginated_response()" or no?

Comment (Feazom) [reply open_schools_platform/organization_management/organizations/views.py:301]: like [here](https://github.com/LamArt/open-schools-platform/blob/dev/open_schools_platform/organization_management/circles/views.py#L76)

**Review by Feazom — APPROVED** [review]

**Pull request merged**